clang/tools/libclang/CASUtils.h

```diff
@@ -19,6 +20,16 @@
 namespace clang {
 namespace cas {
 
+struct WrappedCASOptions {
```

Comment:
You shouldn't need a wrapper if it's just one field like this; you could just do 

```
DEFINE_SIMPLE_CONVERSION_FUNCTIONS(CASOptions, CXCASOptions)
```

Reply:
Done.